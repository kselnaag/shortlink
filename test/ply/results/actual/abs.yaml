abs:  # 22.10.2023, 20:04:12:684
  request:
    url: http://localhost:8080/check/abs
    method: GET
    headers: {}
  response:  # 37 ms
    status:
      code: 404
      message: Not Found
    headers:
      cache-control: no-cache
      connection: close
      content-length: '53'
      content-type: application/json; charset=utf-8
    body: |-
      {
        "Body": "404 Not Found",
        "IsResp": true,
        "Mode": "check"
      }
