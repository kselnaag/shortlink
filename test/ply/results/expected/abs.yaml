abs:
  request:
    url: ${baseUrl}/check/abs
    method: GET
    headers: {}
  response:
    status:
      code: 404
      message: Not Found
    headers:
      cache-control: no-cache
      connection: close
      content-length: '53'
      content-type: application/json; charset=utf-8
    body: |-
      {
        "Body": "404 Not Found",
        "IsResp": true,
        "Mode": "check"
      }
